src/main/java/org/openstreetmap/atlas/geography/atlas/items/LineItem.java

```diff
@@ -24,6 +29,8 @@
 {
     private static final long serialVersionUID = -2053566750957119655L;
 
+    private static final Logger logger = LoggerFactory.getLogger(LineItem.class);
```

Comment:
Nit: is this used anywhere? If not, suggest removing.

Reply:
done 👍